pkg/cli/mirror/mirror.go

```diff
@@ -35,6 +35,53 @@ import (
 	"github.com/openshift/oc-mirror/pkg/metadata/storage"
 )
 
+var (
+	mirrorlongDesc = templates.LongDesc(
+		` 
+		Create and publish user-configured mirrors with a declarative configuration input.
+		Accepts an argument defining the destination for the mirrored images using the prefix file:// for a local mirror packed into a 
+		tar archive or docker:// for images to be stream registry to registry without being stored locally. The default docker credentials are 
+		used for authenticating to the registries. The podman location for credentials is also supported as a secondary location.
+
+		When using file mirroring, the --from and --config flags control the location of the images to mirror. The --config flag accepts
+		an imageset configuration file and the --from flag accepts the location of the imageset on disk. The --from input can be passed as a 
+		file or directory, but must contain only one image sequence. The naming convention for an imageset is mirror_seq<sequence number>_<tar count>.tar.
+
+		The location of the workspace used defaults to oc-mirror-workspace in the current directory when publishing image content or using the  to 
+		mirror workflow. If using the file:// destination scheme, the oc-mirror-workspace directory will be located in the directory specified by the argument.
```

Comment:
I'm having trouble deciding what this sentence means - and perhaps it would make more sense once the previous sentence is complete?

Or maybe this one should be combined with the previous one somehow? Are the options to either 1). default to oc-mirror-workspace in the current directory, or to 2). specify another location using the 'file://' scheme?